### ISSUE TITLE
feat(coupled): nonlinear end-to-end inversion through the radiance operator with Laplace posterior

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/plumax#102]
## Problem / Request
Tier IV has a closed-form `(Q, bias)` fusion for *column* observations and a radiance observation operator (`coupled/rtm.py`: `RadianceObservationOperator`, `LinearisedRadianceOperator`), but the two are not joined: nothing inverts L1 radiances end-to-end (`transport → ΔVMR → radiance` nonlinear in `Q`, plus met/background nuisance parameters), and the design's Step 2 row "End-to-end inversion — reuse `assimilation/` with composed forward" and "Posterior covariance — reuse Tier III's modules" are both ☐.

## User Story
> As a user with EMIT L1 radiances and TROPOMI columns over the same source, I want one inversion over `(Q, x₀, ū, θ, c_bg, bias_inst)` that uses the radiance operator for EMIT and the column AK for TROPOMI, with a Laplace posterior, so fusion works at the radiance level.

## Proposed API
```python
# coupled/inversion.py
@dataclass(frozen=True)
class CoupledProblem:
    forward: CoupledForward
    radiance_ops: dict[str, RadianceObservationOperator]   # per instrument observing radiance
    prior: CoupledPrior                                     # Q lognormal, met tight Gaussians, bias per instrument
    observations: dict[str, jax.Array]

def solve_coupled(problem: CoupledProblem, *, method: Literal["lbfgs", "gauss_newton"] = "gauss_newton", laplace: bool = True) -> CoupledResult: ...
@dataclass(frozen=True)
class CoupledResult:  map: dict; covariance: PosteriorCovariance; converged: bool; per_instrument_residuals: dict
```

## Implementation Steps
- [ ] Add `coupled/inversion.py`; cost assembled from `assimilation.cost.build_cost_xi` with a block `B` over the mixed parameter set and per-instrument `R`
- [ ] Radiance instruments go through `RadianceObservationOperator`; column instruments through `column_response` — one residual bundle
- [ ] Solve with `assimilation.solve.run_gauss_newton`; Laplace covariance via `les_fvm.fourdvar.posterior_covariance` generalised to a PyTree control
- [ ] `CoupledResult` exposes a `PerEventPosterior`-compatible marginal on `Q` for Tier V

## Definition of Done
- [ ] Linear-conditional-Gaussian limit: columns only, linear AK → `solve_coupled` equals `fuse_observations` to 1e-6
- [ ] Synthetic-truth recovery stratified by instrument count (1, 2, 3+): fusion benefit quantified, truth inside 95 % CI
- [ ] Cross-instrument hold-out: predict the held-out instrument from the posterior; RMSE within its `R`
- [ ] `make test && make lint && make typecheck` green

## Testing
- [ ] `tests/coupled/test_inversion.py::test_linear_limit_equals_fusion`
- [ ] `tests/coupled/test_inversion.py::test_holdout_instrument` (`slow`)

## Relationships
- Parent (theme epic): #74
- Blocked by: Tier II/III transport backends; RTM optimal-estimation retrieval (#73) for the radiance-side `S_e`
- Blocks: Q(t) OU process, masked-K multi-source
